stringer.go

```diff
@@ -133,9 +137,15 @@ func main() {
 	}
 	g.Printf(")\n")
 
+	// Define custom error type if typed errors are enabled
+	if *typedErrors {
+		g.Printf("\n// InvalidEnumValueError is returned when an invalid string is provided to an enum conversion function\n")
+		g.Printf("var InvalidEnumValueError = errors.New(\"invalid enum value\")\n")
+	}
```

Comment:
For any package that contains multiple generated enums, this will be duplicated. This is why `InvalidEnumValueError` must come from outside the generated code. See here: https://github.com/bincyber/enumer/blob/b69b324ff5b1e5c5620512c539e590da8b3148ad/pkg/enumer/enumer.go#L5